templates/about.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'base_layout.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1WcNN2RlQQrbLSP5ZgcIsAeQAHj5h_yDW6Yd5oBMmU3M{% endblock meta_copydoc %}
+
+{% block meta_description %}Charmhub description{% endblock %}
```

Comment:
Can you take a sentence from the first paragraph that would be better then this.